### ISSUE TITLE
Latex math delimiter

### ISSUE THREAD

**Issue by alexpacini** [benweet/stackedit#710]
Sorry if this have been already asked, but I did not find anything related.
I have seen the default math delimiter is ```$...$```  or ```$$...$$```.
Is it possible to have an easy check box or  drop down menu to select ```\(...\)``` and ```\[...\]```

**Comment by victorel-petrovich** [comment]
... or maybe  for replacing $ with `\$`

**Comment by 5nizza** [comment]
go Settings->Extensions, there is:

![mathjax](https://cloud.githubusercontent.com/assets/928552/7101380/8ce2212e-e057-11e4-9b2d-614e47b1813d.png)

**Comment by chaoxu** [comment]
Now Stackedit is using katex, how can we make it work here?

**Comment by Greedquest** [comment]
Yes CodeReview and Electrical Engineering stack exchange both use `\$` as the inline delimiter, `$$` for multiline. The ability to toggle would be really useful!
